### PR TITLE
afl-persistent 1.4: add patch to fix macos build

### PR DESCRIPTION
**Pull request by mseri**

### CODE REVIEW

**Comment by mseri** [comment]
Failure is
```
+ /Users/mac1000/.opam/opam-init/hooks/sandbox.sh "build" "dune" "build" "-p" "afl-persistent" "-j" "11" (CWD=/Users/mac1000/.opam/4.14.1/.opam-switch/build/afl-persistent.1.4)
Processing  4/10: [afl-persistent: ./config.sh]
+ /Users/mac1000/.opam/opam-init/hooks/sandbox.sh "build" "./config.sh" (CWD=/Users/mac1000/.opam/4.14.1/.opam-switch/build/afl-persistent.1.4)
- ./config.sh: line 17: cannot create temp file for here document: Operation not permitted
[ERROR] The compilation of afl-persistent.1.4 failed at "./config.sh".
```

File: packages/afl-persistent/afl-persistent.1.4/opam [hunk 1]
```diff
@@ -9,6 +9,9 @@ build: [
   [ "dune" "build" "-p" name "-j" jobs ]
   [ "./config.sh" ]
 ]
+patches: [
+  "files/config.patch"
```
Comment (mseri) [inline packages/afl-persistent/afl-persistent.1.4/opam:13]:
```suggestion
  "config.patch"
```

**Comment by mseri** [comment]
But unrelated

File: packages/afl-persistent/afl-persistent.1.4/opam [hunk 2]
```diff
@@ -9,6 +9,9 @@ build: [
   [ "dune" "build" "-p" name "-j" jobs ]
   [ "./config.sh" ]
 ]
+patches: [
+  "config.patch"
+]
```
Comment (hannesm) [inline packages/afl-persistent/afl-persistent.1.4/opam:14]: what is missing here is a section:
```
extra-files: [
  ["config.patch" "md5=<>"]
```

at least with the upcoming change in opam https://github.com/ocaml/opam/pull/5564 this `extra-files` section is necessary...

Comment (mseri) [reply packages/afl-persistent/afl-persistent.1.4/opam:14]: I like this change, thanks for pointing it out. I'll send a PR immediately.

**Pull request merged**